contrib/guix/manifest.scm

```diff
@@ -28,6 +28,7 @@
              (gnu packages bison)
              (gnu packages tls)
              (gnu packages version-control)
+             (guix build-system cmake)
```

Comment:
27779, 27179 - need to check GUIX build succeed

Reply:
dropped both for now. May investigate more which caused the issue